lib/KTable/index.vue

```diff
@@ -182,6 +192,12 @@
 
       const isTableEmpty = computed(() => sortedRows.value.length === 0);
 
+      const isTableScrollable = computed(() => {
```

Comment:
On the initial load, this value is false and the vertical lines beside the sticky columns don't appear, even though the table can be scrolled.

![Screenshot 2025-06AM](https://github.com/user-attachments/assets/872e0d90-2538-4fdd-afe7-1c59c76b79dd)

Reply:
The woes of testing things locally over and over! :D thanks